app/assets/config/manifest.js

```diff
@@ -12,6 +12,7 @@
 //= link views/_cookie-settings.css
 //= link views/_csv_preview.css
 //= link views/_homepage.css
+//= link views/_homepage_new.css
```

Comment:
I'm finding this second commit a bit hard to follow. I thiiiiink what's happening is you're undoing the css styles you made in the first commit in `homepage.css` , and making those changes in a new file called `homepage_new.css` instead, so that you can make use of the @new_design toggle? But then there are also fresh changes in this commit. So I was wondering if you could rework the first two commits to make it easier to follow. 

BUT I'm also confused as to why we're using the toggle here at all. I was under the impression we'd be using it to incrementally roll out the look and feel changes which is going live mid October - where as this PR is to go live on Monday. Plus you've picked the commits from other cards so it's all rolled into one anyhow. Soooo can we just forget about the toggle in this PR?

Huge apologies if I've totally missed the point of what you're doing here Patrick!

Reply:
Ahhh ok so I think you've added this logic (and new homepage css file) because some of the changes form part of the look and feel designs and monica is wanting to include them early, so you're setting the foundations for being able to use the backend toggle later. But I think that will make the roll out of the layout changes for Monday complicated as we'll need to move just these changes out of the the `if` block. 

So if it's ok with you, I think we should remove that logic out of this PR. And we can add it back when/if we need it in subsequent work

Reply:
I've removed the logic to set the new styles, so the new homepage layout will be used when the changes are merged